Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -156,3 +157,25 @@ def test_no_ec_commands(command, args, response, expected_hr, expected_auto_extr
     result = command(client, args)
     assert expected_hr in result[0]  # HR is found in the 1st place in the result of the command
     assert expected_auto_extract == result[3]  # ignore_auto_extract is in the 4th place in the result of the command
+
+
+def test_fetch_incidents(mocker):
+    """Unit test
+    Given
+    - fetch incidents command
+    - command args
+    - command raw response
+    When
+    - mock the parse_date_range.
+    - mock the Client's send_request.
+    Then
+    - run the fetch incidents command using the Client
+    Validate The length of the results.
+    """
+    mocker.patch('ServiceNowv2.parse_date_range', return_value=("2019-02-23 08:14:21", 'never mind'))
+    client = Client('server_url', 'sc_server_url', 'username', 'password', 'verify', 'proxy', 'fetch_time',
+                    'sysparm_query', sysparm_limit=10, timestamp_field='opened_at',
+                    ticket_type='incident', get_attachments=False)
```

Comment:
can we test with `get_attachments=True` as well?

Reply:
we must, otherwise this test does not reproduce the bug and does not cover the issue

Reply:
Yes, i will create another one for this.